kythe/java/com/google/devtools/kythe/platform/java/filemanager/ForwardingStandardJavaFileManager.java

```diff
@@ -297,6 +297,10 @@ private static IllegalStateException propagateInvocationTargetErrorIfPossible(
       String methodName, ReflectiveOperationException error, Class<IOException> declaredType)
       throws IOException {
     if (error instanceof InvocationTargetException) {
+      // Log the exception because the propagated destination may not provide a nice error log.
+      logger.atWarning().withCause(error).log(
```

Comment:
this should be `error.getCause()`.  `InvocationTargetException` is the type used to wrap non-reflective exceptions originating in the underlying method.

Reply:
done